assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -513,9 +565,80 @@ module.exports = elementorModules.ViewModule.extend( {
 			if ( showFooter ) {
 				this.updateFooterText();
 			}
+
+			this.bindHotKeys();
+
+			jQuery.each( this.buttons, ( index, item ) => {
+				jQuery( item )
+					.attr( 'tabindex', 0 )
+					.keypress( ( event ) => {
+						if ( 13 === event.which || 32 === event.which ) {
+							jQuery( item ).trigger( 'click' );
+						}
+					} );
+			} );
```

Comment:
1. Move it to an external method.
2. As @matipojo mentioned above, you don't need a loop and re-creation of the jQuery instances